aperture.go

```diff
@@ -98,15 +98,19 @@ func run() error {
 		return err
 	}
 
+	var etcdClient *clientv3.Client
 	// Initialize our etcd client.
-	etcdClient, err := clientv3.New(clientv3.Config{
-		Endpoints:   []string{cfg.Etcd.Host},
-		DialTimeout: 5 * time.Second,
-		Username:    cfg.Etcd.User,
-		Password:    cfg.Etcd.Password,
-	})
-	if err != nil {
-		return fmt.Errorf("unable to connect to etcd: %v", err)
+	if cfg.Etcd != nil {
```

Comment:
This might be non-nil if someone just comments out the sub keys. Perhaps we need a new top-level option, for example `backend-mode` that is either `etcd` or `lightweight`. That would replace the `staticSecret.Enabled` and `tor.Filestorage` flags.